Ports/pacman/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=pacman
+version=git
+workdir=pacman-master
```

Comment:
These are strings without variable expansions, so they should be in single-quotes.

```suggestion
port='pacman'
version='git'
workdir='pacman-master'
```